client/packages/core/src/instatx.ts

```diff
@@ -6,7 +8,10 @@ type LookupRef = [string, any];
 type Lookup = string;
 export type Op = [Action, EType, Id | LookupRef, Args];
 
-export interface TransactionChunk {
+export interface TransactionChunk<
+  S extends InstantGraph<any, any>,
```

Comment:
nit: would use more verbose variable names. Also surprised about the `any` here.

Reply:
This is a pattern I see a lot in TS. `any` acts as a placeholder, kinda like to `InstantGraph<_, _>`.  We _could_ substitute it with the types the generic param is extending (i.e. `InstantGraph<EntitiesDef, LinksDef<EntitiesDef>>`), but `InstantGraph` already constrains it.

Reply:
Improved the generic param names tho!